Formula/angle-grinder.rb

```diff
@@ -0,0 +1,17 @@
+class AngleGrinder < Formula
+  desc "Slice and dice log files on the command-line"
+  homepage "https://github.com/rcoh/angle-grinder"
+  url "https://github.com/rcoh/angle-grinder/archive/v0.7.3.tar.gz"
+  sha256 "2993c05c927b2e2a7fd5bf5211bcfce715f747c3889c25c351a34a60e021d50a"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix
+  end
+
+  test do
+    (testpath/"logs.txt").write("{\"key\": 5}")
+    system "#{bin}/agrind", "--file", "logs.txt", "* | json | count by key"
```

Comment:
Please `assert_equal` or `assert_match` something here rather than testing only for a zero exit code.

Reply:
Done, thanks.